### ISSUE TITLE
Function needs to be improved

### ISSUE THREAD

**Issue by lwy8** [keepassxreboot/keepassxc#748]
1, I hope in the interface to see the title, user name, password, URL, of course, the custom field can also be displayed
2, I would like to know how many records in each group, and the entire database recorded a total of how many records.
3, some languages, such as the Chinese language interface is not comprehensive enough, some menus are not translated to complete.
4, chromspass plugin, it seems that some problems, KeePassHttp Settings will be prompted many times.
5, in the editing time, the Advanced tab, add additional attributes, if you can use the history of the property field to drop the list of the form appears to be great.
[
![a056bda9-3518-4157-9db0-c7e560adc2d2](https://user-images.githubusercontent.com/28969411/27853960-7576d250-6197-11e7-9929-48a26ed8365a.png)
](url)
In short keePassXC project is great, I will always support it.

**Comment by droidmonkey** [comment]
#1 is a duplicate. Open #2 in its own issue. #3 please join transifex and help translate. #4 needs a new issue, I don't know what you are saying. #5 not sure what you want here, make a new issue.
